tasks/14.html

```diff
@@ -12,6 +12,10 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
+                .global-scope > ul > li:before
```

Comment:
Зачем такая вложенность? Родительские элементы в этом селекторе вообще можно не указывать
```suggestion
                li:before
```